### PR TITLE
Update formula for weighted average retention rate

### PR DESCRIPTION
**Pull request by mataeuxg**
Update formula in the "When examining a Retention report in % / Retention Rate view" section. C_March 1 and C_March 2 were incorrect.

### CODE REVIEW

File: pages/docs/analysis/reports/retention.md [hunk 1]
```diff
@@ -247,9 +247,9 @@ From our sample data, the average value for day 1 retained users is calculated a
 
 #### When examining a Retention report in % / Retention Rate view:
 
-C_March 1 = 500 / 1000 (retention rate for March 1)
+C_March 1 = 500 (retention rate for March 1)
```
Comment (cechoy) [inline pages/docs/analysis/reports/retention.md:250]: Looks like `C_March 1` refers to the number of retained users in March 1. So the description "retention rate for March 1" is wrong. Should be something like "retained users for March 1" or "count of retained users for March 1".

File: pages/docs/analysis/reports/retention.md [hunk 2]
```diff
@@ -247,9 +247,9 @@ From our sample data, the average value for day 1 retained users is calculated a
...
 
-C_March 2 = 400 / 500 (retention rate for March 2)
+C_March 2 = 400 (retention rate for March 2)
```
Comment (cechoy) [inline pages/docs/analysis/reports/retention.md:252]: Same comment here.

**Comment by ranic** [comment]
Closing since it looks like this was fixed here: https://github.com/mixpanel/docs/pull/271

File: pages/docs/analysis/reports/retention.md (hunk 1 above)
Comment (mataeuxg) [reply pages/docs/analysis/reports/retention.md:250]: Sorry Celia. Just getting back to this. You are correct. Wording should be "count of retained users for March_1".

**Pull request closed without merging**